### PR TITLE
Bugfix for PhaseNet label IDs

### PR DESCRIPTION
**Pull request by JanisHe**
Hi Jannes,

when using the original PhaseNet model or diting as base models for transfer learning, the label IDs are wrong. Default labels_ids are {"P": 0, "S": 1, "Noise": 2}, however, PhaseNet and diting have {"Noise": 0, "P": 1, "S": 2}. Therefore, transfer learning was not possible since the ids did not depend on the given labels by the loaded model. I rewrote the method "_colums_to_dict_and_labels" and added an argument model_labels to get the correct dictionary and label IDs.

Cheers,
Janis

### CODE REVIEW

File: tests/test_generate.py [hunk 1]
```diff
@@ -1051,22 +1051,32 @@ def check_labels_against_ids(labeller, ids, labels):
         labeller(state_dict)
 
 
-def test_colums_to_dict_and_labels():
+@pytest.mark.parametrize(
```
Comment (yetinam) [inline tests/test_generate.py:1054]: This test should also test the case that `model_labels` is None.

File: seisbench/generate/labeling.py [hunk 2]
```diff
@@ -131,16 +131,19 @@ class PickLabeller(SupervisedLabeller, ABC):
     :param kwargs: Kwargs are passed to the SupervisedLabeller superclass
```
Comment (yetinam) [inline seisbench/generate/labeling.py:131]: Please add a docstring for `model_label`. And as I've just spotted that the one for `noise_column` is missing, would you mind adding that too?

**Comment by yetinam** [comment]
Hi @JanisHe ,
thanks a lot for the contribution. Indeed, it's nice to add this flexibility here. I've left a few minor that I'd ask you to address.

I'm not sure I'd call it a bug but it's definitely good to add this functionality. However, that's why I'd prefer merging it into `main`. Would you mind switching the base branch and rebasing your PR?

**Comment by JanisHe** [comment]
Hi Jannes,
I just add your points and then we can merge this to `main`

**Comment by yetinam** [comment]
Hi @JanisHe ,
I'm looking at this now and modifying a few details. I'll hopefully merge it tomorrow.

**Comment by yetinam** [comment]
I've merged the branch now. Thanks a lot for your contribution!
